saleor/graphql/account/schema.py

```diff
@@ -44,7 +45,9 @@ class Meta:
 class AccountQueries(graphene.ObjectType):
     address_validator = graphene.Field(
         AddressValidationData,
-        input=graphene.Argument(AddressValidationInput, required=True),
+        country_code=graphene.Argument(CountryCodeEnum, required=False),
```

Comment:
Isn't this one actually required?

Reply:
I thought the same, but looking at one of the tests, no, it is not
https://github.com/mirumee/saleor/pull/4125/files#diff-14900e464e1e3ba1cd78619e9d93f1e9R1348
Right now we are trying to resolve by IP if the country code is not provided.